tests/src/test/java/alluxio/testutils/LocalAlluxioClusterResource.java

```diff
@@ -284,6 +286,17 @@ public void evaluate() throws Throwable {
             } else {
               resetCluster(fsm);
             }
+
+            // Reset any properties that have changed
```

Comment:
Unfortunately, it's not quite the same as using the `LocalAlluxioResource.Config` annotation, but for any test that sets `ServerConfiguration` duing one of the tests, it's desirable to put the config after the test back to the state it was in before the test failed.

There's no good way to track the set of changes from a specific time, so I just take a copy of the config before

We can't just call `ServerConfiguration.reset()` because that will unset the port numbers for the Alluxio cluster.